### PR TITLE
landscape.io stuff

### PR DESCRIPTION
**Pull request by Cadair**
Seeing how http://landscape.io has just enabled experimental PR support, I thought now would be a good time to configure it and fix some of the more glaring issues it finds.

### CODE REVIEW

**Comment by ayshih** [comment]
Are cyclic imports something we can avoid??

File: sunpy/instr/rhessi.py [hunk 1]
```diff
@@ -371,23 +371,23 @@ def backprojection(calibrated_event_list, pixel_size=(1.,1.) * u.arcsec, image_d
...
+
     #find out what detectors were used
     det_index_mask = afits[1].data.field('det_index_mask')[0]
     detector_list = (np.arange(9)+1) * np.array(det_index_mask)
     for detector in detector_list:
         if detector > 0:
             image = image + _backproject(calibrated_event_list, detector=detector, pixel_size=pixel_size.value
-										 , image_dim=image_dim.value)
-    
+                                         , image_dim=image_dim.value)
```
Comment (dpshelio) [inline sunpy/instr/rhessi.py]: is it normal to start a line with a coma?

File: sunpy/time/time.py [hunk 2]
```diff
@@ -214,7 +214,7 @@ def is_time(time_string, time_format=''):
     time_string : [ int, float, time_string, datetime ]
         Date to parse which can be either time_string, int, datetime object.
     time_format : [ basestring, utime, datetime ]
-	Specifies the format user has provided the time_string in.
+    Specifies the format user has provided the time_string in.
```
Comment (dpshelio) [inline sunpy/time/time.py]: Why the indentation of this one doesn't match the explanation from the `time_string` above?

**Comment by dpshelio** [comment]
Is this bot commenting on excess? or it's because it's still experimental?

**Pull request closed without merging**